.github/workflows/ci.yml

```diff
@@ -18,22 +18,26 @@ jobs:
           - os: windows-latest
             target: Windows
 
+    env:
+      # Take care when modifying this field; it's parsed naïvely by a script.
+      RUSTFLAGS: --deny warnings
```

Comment:
Can we just move this entirely to the `build-all.sh` script? I imagine we only need to deny warnings for the build step, the other `cargo` steps in CI should be fine either way.